experimenter/experimenter/experiments/models.py

```diff
@@ -676,6 +676,34 @@ def results_ready(self):
             )
             return datetime.date.today() >= resultsReadyDate
 
+    @property
+    def has_displayable_results(self):
+        # True if self.results_data has weekly or overall results
+        if self.results_data and "v2" in self.results_data:
+            results_data = self.results_data["v2"]
+            if "overall" in results_data:
+                overall_results = results_data["overall"]
+                if "enrollments" in overall_results:
+                    enrollments = overall_results["enrollments"]
+                    if "all" in enrollments and enrollments["all"] is not None:
+                        return True
+
+            if "weekly" in results_data:
+                weekly_results = results_data["weekly"]
+                if "enrollments" in weekly_results:
+                    enrollments = weekly_results["enrollments"]
+                    if "all" in enrollments and enrollments["all"] is not None:
+                        return True
+
+        return False
```

Comment:
Oooof so many nested if's. But I can't think of a better way to do this, and you've made it super readable

Reply:
we can do this 
```python
    if self.results_data and "v2" in self.results_data:
        results_data = self.results_data["v2"]
        for result_type in ["overall", "weekly"]:
            if result_type in results_data:
                enrollments = results_data[result_type].get("enrollments", {}).get("all")
                if enrollments is not None:
                    return True
    return False

Reply:
We know what the data structure is right? We could do something like this, where we attempt to check for  the fields and if they return None, we assume there is not displayable results. 

```
while True:
            # Check overall results
            try:
                [item for item in self.results_data["v2"]["overall"]["enrollments"]["all"]]
            except (TypeError, KeyError) as e:
                pass
            else:
                return True
            # Check weekly results
            try:
                [item for item in self.results_data["v2"]["weekly"]["enrollments"]["all"]]
            except (TypeError, KeyError) as e:
                pass
            else:
                return True

            return False
```